### ISSUE TITLE
Command priority out of range

### ISSUE THREAD

**Issue by jcansdale** [madskristensen/CommandTableInfo#1]
Any idea why the command priority is so high? If I use this in my own .vsct file, I get a command priority out of range error. 😕 

![image](https://user-images.githubusercontent.com/11719160/41856439-04146b00-788d-11e8-8fcb-92efd5932fa4.png)

**Comment by madskristensen** [comment]
No, I don't know how that is done. Perhaps @dgriffen knows how to add such a big number to the VSCT or some other mechaniesm?

**Comment by ghost** [comment]
I'm not sure. That priority might be hardcoded into vs and not in a vsct file.

**Comment by jcansdale** [comment]
@dgriffen I wondered if it might be some kind of flag rather than an actual priority?

**Comment by ghost** [comment]
@jcansdale I wasn't able to find anything on it.

**Comment by madskristensen** [comment]
No longer showing priority
